lib/nest-away-accessory

```diff
@@ -0,0 +1,109 @@
+/**
```

Comment:
where did this file come from?

Reply:
I’m testing something out 
https://github.com/chrisjshull/homebridge-nest/issues/64